src/views/v2/month/calendar-body/day.php

```diff
@@ -103,6 +103,7 @@ class="tribe-events-calendar-month__mobile-events-icon tribe-events-calendar-mon
 				aria-label="<?php echo esc_attr( $has_featured_events_label ); ?>"
 				title="<?php echo esc_attr( $has_featured_events_label ); ?>"
 			>
+				<?php $this->template( 'components/icons/featured', [ 'classes' => [ 'tribe-events-calendar-month__mobile-events-icon-svg--featured' ] ] ); ?>
```

Comment:
we have the modifier above already in `tribe-events-calendar-month__mobile-events-icon--featured`, so we don't need the modifier here. instead, let's rely on the above modifier

```suggestion
				<?php $this->template( 'components/icons/featured', [ 'classes' => [ 'tribe-events-calendar-month__mobile-events-icon-svg' ] ] ); ?>
```